carissa_palu/Amoamokofecatwalk.js

```diff
@@ -0,0 +1,74 @@
+console.log('AmoamokofeCat!');
+
+var AmoamokofeCat = document.getElementById('AmoamokofeCat')[0];
```

Comment:
Array access at the end of getElementById() will give you an error because the method does not return an array...
Have you run this code and checked the JS console for errors?